launcher/Application.cpp

```diff
@@ -1733,3 +1740,14 @@ void Application::triggerUpdateCheck()
         qDebug() << "Updater not available.";
     }
 }
+
+void Application::addImportUrl(QString const& url)
```

Comment:
Turn this into a function that returns a normalized URL instead and use it when parsing args and when parsing the import command.

That way we can avoid the duplicated code above in L1087-L1090